tools/exporter/exporter.py

```diff
@@ -16,7 +16,8 @@
 import pytz
 import shutil
 from tzlocal import get_localzone
-import csvExporter as csv
+import csvExporter as csv_e
```

Comment:
Can this be made more readable? csv_e is cryptic. Why is unicodecsv needed? Can it be moved into csvExporter instead?